Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -20,160 +20,91 @@
 
 namespace DiscIO
 {
-namespace DiscScrubber
-{
 #define CLUSTER_SIZE 0x8000
```

Comment:
This was going to be changed in a follow-up PR with a bunch of other minor things that weren't particularly related to the class structure itself.